src/main/webapp/assets/modules/contest-detail/contestDetailDirective.js

```diff
@@ -92,7 +93,13 @@ techlooper
     return {
       restrict: "E",
       replace: true,
-      templateUrl: "modules/contest-detail/contestDetailReviewSubmission.html"
+      templateUrl: "modules/contest-detail/contestDetailReviewSubmission.html",
+      link: function (scope, element, attr, ctrl){
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.